definitions/divorce/json/CaseTypeTab.json

```diff
@@ -13,68 +13,68 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "DIVORCE",
     "Channel": "CaseWorker",
-    "TabID": "solicitorDetails",
-    "TabLabel": "Solicitor",
+    "TabID": "petitionDetails",
+    "TabLabel": "Petition",
     "TabDisplayOrder": 2,
     "CaseFieldID": "PetitionerSolicitorName",
-    "TabFieldDisplayOrder": 1
+    "TabFieldDisplayOrder": 22
```

Comment:
Ordering of fields by TabFieldDisplayOrder is a bit crap for now, given this is a large PR I'll try to do it after this is merged